src/compiler/scala/tools/nsc/typechecker/Namers.scala

```diff
@@ -582,20 +582,24 @@ trait Namers extends MethodSynthesis {
         }
       }
 
-      def noDuplicates(names: List[Name], check: DuplicatesErrorKinds.Value) {
-        def loop(xs: List[Name]): Unit = xs match {
+      selectors foreach checkSelector
+
+      def noDuplicates(): Unit = {
+        @inline def isRename(hd: ImportSelector): Boolean =
```

Comment:
In `2.13.x`, this is a method of the `ImportSelector` class, but I do not want to change any API in `2.12.x`.